tools/buildDocfxPreviewExtension.ps1

```diff
@@ -0,0 +1,44 @@
+param(
+    [string] $command = "package",
+    [string] $version
+)
+
+################################################################################################
+# Usage:
+# Run buildDocfxPreviewExtension.ps1
+#   [-command Command]: package or publish, default to be package
+#   [-version Version]: Publish version which should be newer than published version
+################################################################################################
+$ErrorActionPreference = 'Stop'
+$scriptPath = $MyInvocation.MyCommand.Path
+$scriptHome = Split-Path $scriptPath
+$configuration = "Release"
+$framework = "net46"
+$HttpServiceFolder = "DfmHttpService"
+$extensionFolder = "VscPreviewExtension"
+
+function ProcessLastExitCode {
+    param($exitCode, $msg)
+    if ($exitCode -ne 0) {
+        Write-Error "$msg, exit code: $exitCode"
+        Pop-Location
```

Comment:
ok. I reviewed the function first... missing the later push

---
In reply to: [109084783](https://github.com/dotnet/docfx/pull/1511#discussion_r109084783) [](ancestors = 109084783)